### PR TITLE
Issue 200 simplify merge

### PR DESCRIPTION
**Pull request by gkellogg**
I believe this handles the various merge simplifications we discussed.

### CODE REVIEW

File: metadata/index.html [hunk 1]
```diff
@@ -737,7 +735,7 @@
             <dt id="table-notes"><code>notes</code></dt>
             <dd>
               <p>
-                An <a>array property</a> that provides an array of objects representing arbitrary annotations on the annotated tabular data model. This specification does not place any constraints on the structure of these objects.
+                A <a>common property</a> that provides one or more objects representing arbitrary annotations on the annotated tabular data model. This specification does not place any constraints on the structure of these objects.
```
Comment (JeniT) [inline metadata/index.html]: It feels a bit weird to call this a common property, as common properties can go on any type of description object (table group, table, column etc etc). I think this should stay being an array property.

File: metadata/index.html [hunk 2]
```diff
@@ -1920,42 +1918,28 @@
...
-        <li>If the property is an <a>atomic property</a>, then if the property only accepts single values, the value from A overrides that from B; otherwise the result is an array of values: those from A followed by those from B that were not already a value in A.</li>
+        <li>If the property is an <a>object property</a> the objects are merged as described here.</li>
+        <li>If the property is a <a>natural language property</a>, the result is an object whose properties are language codes and where the values of those properties are arrays. The arrays MUST provide the values from A followed by those from B that were not already a value in A. If a value exists in <code>und</code> and any other language the value from <code>und</code> MUST be removed, ensuring that the <code>und</code> property is removed if it's value becomes an empty array.</li>
         <li>If the property is a <a>common property</a>, the result is an array containing values from A followed by values from B not already in A.</li>
```
Comment (JeniT) [inline metadata/index.html]: Do we need to say something here about how object equality is assessed, when a common property has an object as a value? Is it deep equality?

**Comment by JeniT** [comment]
@gkellogg see the couple of comments above, otherwise looking really good

File: metadata/index.html (hunk 1 above)
Comment (gkellogg) [inline metadata/index.html]: Good point, I'll revert this. Probably just do supimple append without comparing for duplicates, which would be "garbage in" anyway"

File: metadata/index.html (hunk 2 above)
Comment (gkellogg) [inline metadata/index.html]: No, and that would be unfortunate. As with notes, just do simple append without looking for duplicates.

**Comment by JeniT** [comment]
@gkellogg I've amended the example of the merge fairly extensively, so you will want to check it. I'm going to create a couple of issues as well to resolve separately.

**Pull request merged**